pkg/webhook/acceleratednetwork/add.go

```diff
@@ -27,5 +28,10 @@ func AddToManager(mgr manager.Manager) (*extensionswebhook.Webhook, error) {
 			{Obj: &appsv1.DaemonSet{}},
 		},
 		Mutator: NewMutator(mgr, logger),
+		ObjectSelector: &metav1.LabelSelector{
+			MatchLabels: map[string]string{
+				"k8s-app": "calico-node",
```

Comment:
I thought we would have a constant for `"k8s-app"`, but it seems that we do not (yet) have one. fine for me (for now).